### ISSUE TITLE
Error not applicable with Crypto :: Conversion rate cannot be 0 or 1

### ISSUE THREAD

**Issue by frewper** [frappe/erpnext#41559]
### Information about bug

We also use crypto as a currency. 

One of the crypto we are dealing with  is USDC which is tied to the US dollar and the value is exactly 1. i.e. 1 USDC = 1 USD.

When ever I create a transaction using this crypto, ex Purchase Invoice, When I try to save the transaction, it gives an error

"Conversion rate cannot be 0 or 1"

This should not be happening as this is a valid transacstion in the real world. 

Hence this error check for 0 or 1 (in this case 1) is invalid and needs to be corrected.

### Module

accounts

### Version

ERPNext: v14.64.0 (version-14)

Frappe Framework: v14.66.3 (version-14)

### Installation method

manual install

### Relevant log output / Stack trace / Full Error Message.

```shell
Conversion rate cannot be 0 or 1
```

**Comment by Nihantra-Patel** [comment]
Because validation already set:

https://github.com/frappe/erpnext/blob/dfaca932920efb4083ef4071a074095b1e25738d/erpnext/controllers/accounts_controller.py#L2325-L2334

It's responsible for checking the validity of a conversion rate between currencies. Here's what it does:

1. It first gets the default currency of the company where the transaction is occurring.
2. Then it checks if there is a default currency set for the company. If not, it throws an error asking the user to enter the default currency.
3. Next, it evaluates three conditions:
   a. If the currency of the transaction matches the default currency and the conversion rate is not 1, or if there's no conversion rate specified.
   b. If the currency of the transaction doesn't match the default currency and the conversion rate is 1.
   If any of these conditions are met, it throws an error stating that the conversion rate cannot be 0 or 1.

In simpler terms, the code ensures that the conversion rate between currencies is valid. It checks if the conversion rate is neither 0 nor 1, and it makes sure that the conversion rate is specified when the currencies involved are different from the default currency of the company.

**Comment by nabinhait** [comment]
As this is applicable to a very rare use-case, we have decided to keep the validation. You can bypass the validation by overriding the function, closing the issue.
